test/browser-specific/bundle.spec.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+
+it('should build a non-broken bundle for AMD', function (done) {
+  var bundle = path.join(process.cwd(), 'mocha.js');
+  fs.readFile(bundle, function (err, data) {
+    if (err) { return done(err); }
+
+    var content = data.toString();
```

Comment:
it might be more straightforward to use `expect(content).not.to.match(/define.amd/)` or something like this